### PR TITLE
Change password reset message

### PR DESCRIPTION
**Pull request by sigberto**
Previous message asked user to log in, but they were already logged in.
Chose to simply change message instead of immediate redirect to dashboard as that seems
to be the standard after a brief review of different sites.

### CODE REVIEW

**Comment by sigberto** [comment]
@stvstnfrd want to take a quick look?

**Comment by sigberto** [comment]
@stvstnfrd fixed in the random pep8 issue. everything related to my actual pr passed.

File: lms/templates/registration/password_reset_complete.html [hunk 1]
```diff
@@ -45,9 +45,7 @@ <h1 class="title">
   <section class="passwordreset container">
     {% block content %}
     <section role="main" class="content">
-      {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
-      Your password has been set.  You may go ahead and {{ link_start }}log in{{ link_end }} now.
-      {% endblocktrans %}
+      {% trans "Your password has been changed successfully. You are now logged in." %}
```
Comment (stvstnfrd) [inline lms/templates/registration/password_reset_complete.html]: We don't want to change the first part of the message,
`Your password has been set.`;
we should only be changing the second,
`s/You may go ahead and {{ link_start }}log in{{ link_end }} now./You are now logged in.`.

1. This is what the story requested.
2. This minimizes the fork from upstream.

File: lms/djangoapps/student_account/test/test_views.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def test_password_change(self):
             follow=True
         )
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, "Your password has been set.")
+        self.assertContains(response, "Your password has been changed successfully")
```
Comment (stvstnfrd) [inline lms/djangoapps/student_account/test/test_views.py]: Revert this change, per below.

**Comment by sigberto** [comment]
@stvstnfrd all one
